testing/zerotier-one/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Kyle Parisi <kyleparisi@gmail.com>
+# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+pkgname=zerotier-one
+pkgver=1.1.14
+pkgrel=0
+pkgdesc="ZeroTier One allows systems to join and participate in ZeroTier virtual networks."
+url="https://www.zerotier.com/"
+arch="all"
+license="GPL3"
+depends=""
+depends_doc=""
+makedepends="nodejs linux-headers"
+install=""
+subpackages="$pkgname-doc"
+source="
+    $pkgname-$pkgver.tar.gz::https://github.com/zerotier/ZeroTierOne/archive/${pkgver}.tar.gz
+    zerotier-one.initd
+    "
+
+builddir="$srcdir"/ZeroTierOne-$pkgver
+build() {
+    cd "$builddir"
+    sed -i.bak s:/bash:/sh:g ./doc/build.sh
+    make || return 1
+}
+
+package() {
+
+    cd "$builddir"
+
+    # setup init script
+    install -m755 -D "$srcdir"/$pkgname.initd \
+            "$pkgdir"/etc/init.d/$pkgname || return 1
+
+    # link all the binaries
```

Comment:
This comment is kinda useless and also wrong, `make install` does not _link_ binaries, but installs (copies) them to `DESTDIR`.